assets/js/app.js

```diff
@@ -24,7 +19,7 @@ document.addEventListener("DOMContentLoaded", TransientControl.setup);
 //
 import "phoenix_html";
 import { Socket } from "phoenix";
-import { LiveSocket, Browser } from "phoenix_live_view";
```

Comment:
Browser is not exported in [phoenix_liveview js ](https://github.com/phoenixframework/phoenix_live_view/blob/f778e5bb1a4b0a29f8d688bbc6c0b7182dea51ca/priv/static/phoenix_live_view.esm.js#L4934)